source/game/field/obj/ObjectColMgr.cc

```diff
@@ -0,0 +1,49 @@
+#include "ObjectColMgr.hh"
+
+#include "game/system/ResourceManager.hh"
+
+namespace Field {
+
+void *ObjectColMgr::LoadFile(const char *filename) {
+    auto *resMgr = System::ResourceManager::Instance();
+    return resMgr->getFile(filename, nullptr, System::ArchiveId::Course);
+}
+
+/// @addr{0x807C4CE8}
+ObjectColMgr::ObjectColMgr(const char *filename) : m_kclScale(1.0f) {
+    // In the base game, this file is loaded in each GeoObjectKCL and passed into
+    // this function. It's simpler to just keep it here.
+    void *file = LoadFile(filename);
+    m_data = new KColData(file);
+    m_mtx = EGG::Matrix34f::ident;
+    m_mtxInv = EGG::Matrix34f::ident;
```

Comment:
`m_mtx`, `m_mtxInv`, and `_68` should all be initialized in an initializer list to avoid double-assignment.